website/static/js/fangorn.js

```diff
@@ -2230,6 +2255,21 @@ tbOptions = {
     uploads : true,         // Turns dropzone on/off.
     columnTitles : _fangornColumnTitles,
     resolveRows : _fangornResolveRows,
+    // Move obj.data.attributes contents to obj.data , for WB v1 API
+    lazyLoadPreprocess: function(obj) {
+        if (!$.isArray(obj.data)) {
+            $.extend(true, obj.data, obj.data.attributes);
+            delete obj.data.attributes;
```

Comment:
Time for me to be parannoying (paranoid + annoying).  If there is a `data.attributes.attributes` attribute, it'll be deleted by this code.  I'd recommend saving `data.attributes`, then deleting it from `data`, then extending `data` with it.  You might also want to break it out into a small anonymous function so that you can reuse it in the for loop below.